### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by josheewa** [jcgter777/UniChatBots#18]
@Legend-of-iPhoenix can you check https://github.com/jcgter777/UniChatBots/blob/master/FriendlyBOT.js 
I added 
o/ someone
bot: someone: how are you today, sir?

I need you to check it

**Comment by Legend-of-iPhoenix** [comment]
Yes, that is *technically* valid syntax, but it's bad valid syntax. I'll help you fix it tomorrow:)
